lxd/instance_get.go

```diff
@@ -124,6 +126,9 @@ func instanceGet(d *Daemon, r *http.Request) response.Response {
 		recursion = 0
 	}
 
+	// Detect if we want to also returns entitlements for each instance.
+	withEntitlements := request.QueryParam(r, "with-entitlements") == "true"
```

Comment:
We might want to use `shared.IsTrue` here to also capture `yes`, and `1`, or anything capitalised.

Reply:
fixed